spec/integration/indexer/deletion_spec.rb

```diff
@@ -18,6 +18,20 @@
       expect_document_missing_in_rummager(id: "/an-example-page", index: "government_test")
     end
 
+    it "removes a document from the index, explicitly giving the type" do
```

Comment:
Please can you link to https://github.com/alphagov/search-api/blob/main/lib/index.rb#L215 in commit